packages/svelte/src/compiler/phases/3-transform/client/visitors/RegularElement.js

```diff
@@ -227,7 +227,7 @@ export function RegularElement(node, context) {
 			node_id,
 			attributes_id,
 			(node.metadata.svg || node.metadata.mathml || is_custom_element_node(node)) && b.true,
-			node.name.includes('-') && b.true,
```

Comment:
There are also other places where we use this check...could you replace those places too?

https://github.com/sveltejs/svelte/blob/09510c8c5c0888578446781e54f2229428e21bfc/packages/svelte/src/html-tree-validation.js#L187

https://github.com/sveltejs/svelte/blob/09510c8c5c0888578446781e54f2229428e21bfc/packages/svelte/src/html-tree-validation.js#L156

https://github.com/sveltejs/svelte/blob/09510c8c5c0888578446781e54f2229428e21bfc/packages/svelte/src/html-tree-validation.js#L147